REPO.bazel

```diff
@@ -0,0 +1 @@
+ignore_directories(["**/node_modules", "**/**/node_modules"])
```

Comment:
_Nice_

Reply:
Yeah, nice feature that I think came in with Bazel 8